integration_tests/e2e/overview/timeline.cy.ts

```diff
@@ -0,0 +1,104 @@
+import Page from '../../pages/page'
+import OverviewPage from '../../pages/overview/OverviewPage'
+import TimelinePage from '../../pages/overview/TimelinePage'
+
+context('Prisoner Overview page - Timeline tab', () => {
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubSignInAsUserWithEditAuthority')
+    cy.task('stubAuthUser')
+    cy.task('stubGetHeaderComponent')
+    cy.task('stubGetFooterComponent')
+    cy.task('stubPrisonerList')
+    cy.task('stubCiagInductionList')
+    cy.task('stubActionPlansList')
+    cy.task('getPrisonerById', 'G6115VJ')
+    cy.task('getPrisonerById', 'H4115SD')
+    cy.task('getActionPlan', 'G6115VJ')
+    cy.task('getActionPlan', 'H4115SD')
+    cy.task('stubGetShortQuestionSetCiagProfile')
+    cy.task('stubLearnerProfile', 'G6115VJ')
+    cy.task('stubLearnerProfile', 'H4115SD')
+    cy.task('stubLearnerEducation', 'G6115VJ')
+    cy.task('stubLearnerEducation', 'H4115SD')
+    cy.task('stubGetAllPrisons')
+  })
+
+  it('should display timeline given prisoner where one goal was created as part of their initial action plan', () => {
```

Comment:
The first 2 are essentially what the bug in RR-473 was all about - this first one is where the prisoner had their action plan created with 1 goal, and we expect to see "Goal created" (singular) on the timeline